atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -476,6 +476,16 @@ def add_log_type(output):
         # Parent of squashed built image is base image
         image_id = self.workflow.builder.image_id
         parent_id = self.workflow.base_image_inspect['Id']
+
+        # Read config from the registry using v2 schema 2 digest
+        config = self.workflow.push_conf.docker_registries[0].config
+
+        # We don't need container_config section
+        for k in config.keys():
+            if k.lower() == 'container_config':
+                del config[k]
+                break
```

Comment:
Stupid observation, but does this modify the object that is stored in `self.workflow.push_conf.docker_registries[0]`?